GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/RO_BDB_ApolloRevamp.cfg

```diff
@@ -0,0 +1,1358 @@
+	@mass = 0.311	//active CBM Type II
+	%maxTemp = 1973.15
+	%skinMaxTemp = 3600
+	
+	@MODULE[ModuleDockingNode]
+	{
+		%acquireForce = 0.5 // 2
+		%acquireMinFwdDot = 0.8 // 0.7
+		%acquireminRollDot = -3.40282347E+38
+		%acquireRange = 0.25 // 0.5
+		%acquireTorque = 0.5 // 2.0
+		%captureMaxRvel = 0.1 // 0.3
+		%captureMinFwdDot = 0.998
+		%captureMinRollDot = -3.40282347E+38
+		%captureRange = 0.05 // 0.06
+		%minDistanceToReEngage = 0.25 // 1.0
+		%undockEjectionForce = 0.1 // 10
+	}
+}
+
+@PART[bluedog_Apollo_CADS_passive]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.553
+	
+	@title = CADS Docking Port (Passive)
+	%manufacturer = RKK Energiya
+	@description = Common Androgynous Docking System, passive version intended to mate with active counterparts. Useful for permanent fixtures that won't have to form the active docking partner, such as berthing ports on station modules.
+	@mass = 0.2	//passive CBM
+	%maxTemp = 1973.15
+	%skinMaxTemp = 3600
+	
+	@MODULE[ModuleDockingNode]
+	{
+		%acquireForce = 0.5 // 2
+		%acquireMinFwdDot = 0.8 // 0.7
+		%acquireminRollDot = -3.40282347E+38
+		%acquireRange = 0.25 // 0.5
+		%acquireTorque = 0.5 // 2.0
+		%captureMaxRvel = 0.1 // 0.3
+		%captureMinFwdDot = 0.998
+		%captureMinRollDot = -3.40282347E+38
+		%captureRange = 0.05 // 0.06
+		%minDistanceToReEngage = 0.25 // 1.0
+		%undockEjectionForce = 0.1 // 10
+	}
+}
+
+@PART[bluedog_Apollo_DockingSpotlight]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.553
+	
+	@title = Apollo Docking Floodlight
+	%manufacturer = Rockwell International
+	@description = Essentially a pen light glued to a flap, this light (traditionally mounted to the decoupler ring on the Apollo CSM) swings out to provide illumination for docking/berthing operations.
+	@mass = 0.002
+	%maxTemp = 673.15
+	%skinMaxTemp = 773.15
+	
+}
+
+@PART[bluedog_Apollo_EVAFloodlight]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.553
+	
+	@title = Apollo Docking Floodlight
+	%manufacturer = Rockwell International
+	@description = Essentially a pen light on the end of a straw, this light (traditionally mounted to the decoupler ring on the Apollo CSM) swings out to provide illumination to aid kerbonauts on EVAs.
+	@mass = 0.005
+	%maxTemp = 673.15
+	%skinMaxTemp = 773.15
+	
+}
+
+@PART[bluedog_Apollo_EngineMount]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.553
+	
+	@title = Apollo SM Engine Mount
+	%manufacturer = Rockwell International
+	@description = Mounting plate for attaching a single engine to Apollo Service Module. Includes a node just below the top rim for attaching the Saturn SLA as well as a mounting point for the high gain antenna system.
+	@mass = 0.15
+	%maxTemp = 673.15
+	%skinMaxTemp = 773.15
+	
+}
+
+@PART[bluedog_Apollo_RCS_3Way]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	//%useRcsConfig = RCSBlock
```

Comment:
Why aren't we using the generic RCS config tool here, and are writing the RCS config directly?  @Capkirk123 , if you have any insight on configuring RCS blocks...